bin/build-native.sh

```diff
@@ -27,5 +27,5 @@ rm -r $OUTDIR/* || true
 platformio pkg update --environment native || platformioFailed
 pio run --environment native || platformioFailed
 cp .pio/build/native/program "$OUTDIR/meshtasticd_linux_$(uname -m)"
-cp bin/device-install.* $OUTDIR
-cp bin/device-update.* $OUTDIR
+cp bin/native-install.* $OUTDIR
+cp bin/native-run.* $OUTDIR
```

Comment:
Native install is good, but native-run has pio build commands and doesn't really make sense for the release assetts

Reply:
👌, only added that in for cases where people intend to test the build before installing.